src/modules/accommodations/components/reviews/review-card.tsx

```diff
@@ -77,12 +90,12 @@ export function ReviewCard({ review }: ReviewCardProps) {
                 {userName}
               </Typography>
               <Typography variant="body2" color="textSecondary">
-                {userCountry}
+                {userInfo}
               </Typography>
             </Box>
           </Box>
           <Typography variant="body2" color="textSecondary">
-            {review.createdAt}
+            Commented in {review.createdAt.split(' ')[0]}
```

Comment:
We decided to use user friendly more readable date to display when the user joined StayHub. But here we are again using non-readable date. Doesn't it make more sense to use the same date structure here too?) Please create that date formatter as a util and reuse it

Reply:
Also please use time-related util & constants